services/meta/client.go

```diff
@@ -100,6 +100,12 @@ func (c *Client) Close() error {
 	return nil
 }
 
+func (c *Client) Data() *Data {
```

Comment:
No reason not to? I was going to go through the client and change everywhere that was accessing `data` to just use this. Since they're all locking around it.